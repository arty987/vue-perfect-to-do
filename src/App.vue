<template>
  <AppHeader/>
  <AppMain/>
</template>

<script>
import AppHeader from "@/components/AppHeader.vue";
import AppMain from "@/components/AppMain.vue";

export default {
  name: 'App',
  components: {
    AppMain,
    AppHeader
  }
}
</script>

<style>
body {
  margin: 0;
}
</style>
