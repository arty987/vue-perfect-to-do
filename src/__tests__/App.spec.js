import {shallowMount} from '@vue/test-utils'
import App from "@/App.vue";

describe('App.vue', () => {
    it('Should render App', () => {
        const wrapper = shallowMount(App);
        expect(wrapper.element).toMatchSnapshot();
    });
})