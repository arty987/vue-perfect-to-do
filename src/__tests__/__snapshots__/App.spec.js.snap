// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue Should render App 1`] = `<app-content-stub />`;
